database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowStoreEntryComponent.scala

```diff
@@ -15,6 +15,10 @@ trait WorkflowStoreEntryComponent {
 
     def workflowExecutionUuid = column[String]("WORKFLOW_EXECUTION_UUID")
 
+    def workflowType = column[String]("WORKFLOW_TYPE", O.Length(30))
+
+    def workflowTypeVersion = column[Option[String]]("WORKFLOW_TYPE_VERSION")
```

Comment:
Apparently no `O.Length` for `VARCHAR(255)` is the Cromwell Way, cuz that's how everything else is.  :man_shrugging: